app/content/accessibility.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Accessibility statement for the Register of training providers
+---
+
+<!-- markdownlint-disable MD033 -->
+
+This accessibility statement applies to the Register of training providers.
+
+This website is run by the Department for Education. We want as many people as possible to be able to use this website. For example, that means you should be able to:
+
+- change colours, contrast levels and fonts
+- zoom in up to 400% without the text spilling off the screen
+- navigate most of the website using just a keyboard
+- navigate most of the website using speech recognition software
+- listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
+
+We’ve also made the website text as simple as possible to understand.
+
+<a href="https://mcmw.abilitynet.org.uk/" target="_blank" rel="noreferrer nofollow" class="govuk-link">AbilityNet (opens in new tab)</a> has advice on making your device easier to use if you have a disability.
+
+## How accessible this website is
+
+The content is non-accessible for the following reasons:
+
+### Specific error suggestions are not provided for some fields on forms
+
+This can be seen on date fields, if the user inputs an incorrectly formatted date, the error message does not provide a suggestion on how to rectify it. This fails WCAG 2.2 success criterion 3.3.1 Error Identification.
+
+We’re waiting on third parties to assist with remediation so a fix date cannot be provided.
+
+### Unsupported ARIA attributes are present on some form inputs
+
+This may cause issues for users of assistive technology. This fails WCAG 2.2 success criterion 4.1.2 Name, Role, Value.
+
+We’re waiting on third parties to assist with remediation so a fix date cannot be provided.
+
+### While navigating with JAWS, parts of error messages are announced multiple times
+
+This fails WCAG 2.2 success criterion 4.1.2 Name. Role, Value.
+
+We’re waiting on third parties to assist with remediation so a fix date cannot be provided.
+
+## Feedback and contact information
+
+If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:
+
+email <becomingateacher@digital.education.gov.uk>
+
+We’ll consider your request and get back to you in 5 days.
+
+## Reporting accessibility problems with this website
+
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact: <becomingateacher@digital.education.gov.uk> Include ‘Accessibility’ in the subject line of your email.
```

Comment:
Should the email already contain the subject line?

Reply:
I have updated the email address to include the subject.